src/pattern_library/css/blocks/mailing-list-checkbox.css

```diff
@@ -0,0 +1,7 @@
+.mailing-list-checkbox {
+	max-width: var(--form-field-max-width);
+}
+
+.mailing-list-checkbox > p {
+	padding-left: calc(0.5em + var(--space-s));
+}
```

Comment:
a mailing list checkbox seems so specific, 
isn't this just a generic checkbox element (when thinking about how it is a "pattern"?